src/index.html

```diff
@@ -2,16 +2,168 @@
+          >
+        </div>
+        <div class="item">
+          <label for="name">Name:</label>
+          <input
+             type="text"
+             name="name"
+             autocomplete="off"
+             id="name"
+          >
+        </div>
+        <div class="item">
+          <label for="age">How old are You?</label>
+          <input
+             type="number"
+             name="age"
+             min="1"
+             max="100"
+             value="12"
+             id="age"
+
+          >
+        </div>
+        <div class="item">
+          <label for="birth">Full date of birth:</label>
+          <input
+             type="date"
+             name="birth"
+             max="2021-12-01"
+             id="birth"
+          >
+        </div>
+        <div class="item">
+          <label for="agreement">I accept the term of the agreement</label>
+          <input
+             type="checkbox"
+             name="agreement"
+             required
+             id="agreement"
+          >
+        </div>
+
+      </fieldset>
+      <fieldset class="form">
+        <legend>Registration</legend>
+        <div class="item">
+          <label for="email" class="top">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+            id="email"
+          >
+        </div>
+        <div class="item">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="Password"
+            required minlength="5"
+            maxlength="8"
+            id="password"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="form">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="item">
+          <span>Do you love cats?</span>
+            <input type="radio" id="yes"
+              name="yes" value="yes"
+            >
+              <label for="yes">Yes</label>
+            <input type="radio" id="no"
+              name="no" value="no"
+            >
+              <label for="no">No</label>
+        </div>
+        <div class="item">
+          <label for="color">What is your favorite color?</label>
+          <input
+            type="color"
+            name="color"
+            id="color"
+          >
+        </div>
+        <div class="item">
+          <label for="bedtime">What time do you go to bed?</label>
+          <input
+            type="time"
+            name="bedtime"
+            id="bedtime"
+          >
+        </div>
+
+        <div class="item">
+          <label for="cars">What are your favorite brand of cars?</label>
+          <select
+            name="cars"
+            class="selector"
+            multiple
+            id="cars">
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+            <option value="0"></option>
+          </select>
```

Comment:
align opening and closing brackets
when passing values to server better or lowercase them
```suggestion
          <select
            name="cars"
            class="selector"
            multiple
            id="cars"
          >
            <option value="bmw">BMW</option>
            <option value="audi">Audi</option>
            <option value="lada">Lada</option>
            <option value="0"></option>
          </select>
```